mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -0,0 +1,41 @@
+use crate::settings::Settings;
+use anyhow::Result;
+use tracing::info;
```

Comment:
we try to follow this structure for use:

1. std
2. 3rd party crates (anyhow, tracing, ...)
3. 1st party crates (mqtt_broker, ...)
4. local crate (`crate::`)
5 mod declaration (if any)

All five groups should be divided by newline.